package.json

```diff
@@ -22,12 +24,15 @@
     "url": "git@github.com:bjrmatos/jsreport-jade.git"
   },
   "dependencies": {
+    "in-publish": "2.0.0",
     "object-assign": "4.0.1",
     "toner-jade": "1.0.0"
   },
   "devDependencies": {
+    "brace": "0.8.0",
```

Comment:
This looks right to me.

`jsreport-studio-build` uses webpack to bundle everything that extension's UI requires into single `main.js` entry. That one is then used by the jsreport studio. The source files like `brace` or `react` are relevant only during the bundling. 

Why you think it should be in `dependencies`?